src/qt/unlimitedmodel.cpp

```diff
@@ -50,20 +50,34 @@ void UnlimitedModel::Init()
     // Ensure restart flag is unset on client startup
     setRestartRequired(false);
 
+    unsigned int ead = excessiveBlockSize;
```

Comment:
The names have been made more expressive ( tmpExcessiveBlockSize &  tmpMaxGeneratedBlock)